deployment/kustomizations/kustomization.yaml

```diff
@@ -0,0 +1,60 @@
+##### Dependancies used in this kustomization:
+# nginx.ingress
+# external-dns
+# cert-manager
+# kubectl kustomize
+
+#test output like this:
+#kubectl kustomize deployment/kustomizations --enable-helm > deployment/kustomizations/wrenai.kustimized.yaml
+#kubectl create namespace wren
+#kubectl apply -f deployment/kustomizations/wrenai.kustimized.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Namespace for all resources here and in the Helm charts
+namespace: wren
+helmCharts:
+
+  - name: postgresql
+    repo: https://charts.bitnami.com/bitnami
+    version: 14.3.3
+    releaseName: wren-postgresql
+    valuesFile: helm-values_postgresql_14.yaml
+    includeCRDs: true
+    # the Same Namespace
+    namespace: wren
+
+  - name: qdrant
+    repo: https://qdrant.github.io/qdrant-helm
+    version: 0.8.5
+    releaseName: wren-qdrant
+    valuesFile: helm-values-qdrant_0.8.5.yaml
+    includeCRDs: true
+    # The Same Namespace
+    namespace: wren
+
+
+images:
+  - name: ghcr.io/canner/wren-bootstrap
+    newTag: 0.1.4 # Update to new version
+  - name: ghcr.io/canner/wren-engine
+    newTag: 0.4.4 # Update to new version
+  - name: ghcr.io/canner/wren-ui
+    newTag: 0.5.6 # Update to new version
+  - name: ghcr.io/canner/wren-ai-service
+    newTag: 0.3.7 # Update to new version
+
+resources:
+  - base/cm.yaml
+  - base/deploy-wren-ui.yaml
+  - base/deploy-wren-engine.yaml
+  - base/deploy-wren-ai-service.yaml
+  - base/pvc.yaml
+  - base/svc.yaml
+### Modify these examples first and uncomment them:
+  - examples/ingress-wren_example.yaml
+  - examples/certificate-wren_example.yaml
```

Comment:
Should we put comments here? since ingress & certificate are examples provided to users.

Reply:
Sure, np

Reply:
ohh, sorry my bad, I missed the `-`.

Reply:
@qdrddr I thought `ingress` & `certificate` are optional ? If they're optional, maybe put `#` to comment them out ?

Reply:
in k8s I would say these will be used most of the time. Its kind of whole point in having this to be published. And normally with ingress you would need a certificate. So I would say its better to keep it than to remove it.

Reply:
I think for different cloud providers, ingress controllers would be different, for example, we used `aws-load-balancer-controller` on EKS before, but sure, I suppose users could make changes based on their own needs.

I think we could keep it here.

Reply:
@wwwy3y3 thats a good point, different Cloud Providers might utilize different ingress controllers. With on-prem self hosted k8s typically you would see nginx ingress, that I have provided.

supplementing other examples of ingress manifests for other ingress controllers would make this more user friendly.

Though as you can see from the alb specification its basically exactly the same manifest, the only difference is the `ingressClassName: alb`
https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.8/guide/ingress/spec/